:book: add note about configuration in odoo.sh (#316)
itpp-labs/sync-addons: sync/doc/index.rst

* If your Sync projects use webhooks (most likely), be sure that url opens correct database without asking to select one

Odoo.sh
-------

> ``queue_job`` will not work in odoo.sh with workers more than 1. See the details `here <https://github.com/OCA/queue/pull/256#issuecomment-895111832>`__

Reviewer: ```suggestion
``queue_job`` may not work properly in odoo.sh with workers more than 1. See the details `here <https://github.com/OCA/queue/pull/256#issuecomment-895111832>`__
```

Reviewer: не делай ссылку "смотри здесь" -- ссылки вырезаются в магазине приложений. Нужен текст, который понятен в случае исчезновения ссылок.

> For the queue_job work correctly in odoo.sh additional configuration is needed.
> 
> * add these lines to odoo.conf::

Reviewer: ```suggestion
To set up `queue_job` module in odoo.sh  add following lines to `~/.config/odoo.conf` and restart odoo via `odoo-restart` command in Webshell.
```

    [queue_job]
    scheme=https
    port=443

>     host=ODOO_SH_ADDRESS.com

Reviewer: ```suggestion
    host=YOUR_ODOO_ADDRESS.com
```
